src/librbd/io/Dispatcher.h

```diff
@@ -76,7 +76,10 @@ class Dispatcher : public DispatchInterfaceT {
     {
       std::unique_lock locker{m_lock};
       auto it = m_dispatches.find(dispatch_layer);
-      ceph_assert(it != m_dispatches.end());
+      if (it == m_dispatches.end()) {
+	on_finish->complete(0);
```

Comment:
Nit: indentation seems to be mixing tabs/spaces

Reply:
Updated